packages/cryptography/test/unit/Mnemonic.js

```diff
@@ -53,11 +53,27 @@ describe("Mnemonic", function () {
         const expectedKey =
             "302e020100300506032b657004220420853f15aecd22706b105da1d709b4ac05b4906170c2b9c7495dff9af49e1391da";
 
-        const key = await mnemonic.toPrivateKey();
+        const key = await mnemonic.toEd25519PrivateKey();
 
         expect(key.toString()).to.eql(expectedKey);
     });
 
+    it("should produce the expected private key with ecdsa", async function () {
+        const mnemonic = await Mnemonic.fromString(
+            "help please spend actor steel pottery increase rookie trash naive soldier siren"
+        );
+
+        const expectedKey =
+            "f9c31c38697c5bf26645d5521b576ca0369be0dbfec6156ef3c74d0130022dc7";
+
+        let key = await mnemonic.toEcdsaPrivateKey();
+        console.log((await key.derive(-1)).toStringRaw());
+        console.log((await key.derive(0)).toStringRaw());
+        console.log((await key.derive(1)).toStringRaw());
+
+        expect(key.toStringRaw()).to.eql(expectedKey);
+    });
+
```

Comment:
Missing test coverage with customized path.

Reply:
Current this test does not pass, so yea we'll need to figure it out. However, the reason it doesn't pass is what's still confusing to me. I provided the path `44/60/0/0` for ECDSA but I'm still getting different keys. I've tried printing out the different common derivations `-1`, `0`, and `1`, but neither of those produce the expected key. I'm not entirely sure what I'm doing wrong though, so I was hoping another pair of eyes would help here.